### PR TITLE
[SVS] Update XiaoiceSing

### PR DESCRIPTION
**Pull request by A-Quarter-Mile**
Hi @ftshijt, here are updates on Muskits:

- Add slur in svs
- Add pitch_loss and vuv_loss for XiaoiceSing following XiaoiceSing2.

### CODE REVIEW

File: espnet2/gan_svs/espnet_model.py [hunk 1]
```diff
@@ -333,6 +340,8 @@ def collect_feats(
         duration_ruled_phn_lengths: Optional[torch.Tensor] = None,
         duration_syb: Optional[torch.Tensor] = None,
         duration_syb_lengths: Optional[torch.Tensor] = None,
+        slur: Optional[torch.Tensor] = None,
```
Comment (ftshijt) [inline espnet2/gan_svs/espnet_model.py:343]: I guess you also need to add slur preparation in the TEMPLATE/svs1/svs.sh and data.sh ?

File: espnet2/svs/xiaoice/XiaoiceSing.py [hunk 2]
```diff
@@ -455,7 +484,9 @@ def forward(
         midi = midi[:, : midi_lengths.max()]  # for data-parallel
         label = label[:, : label_lengths.max()]  # for data-parallel
         tempo = tempo[:, : tempo_lengths.max()]  # for data-parallel
-
+        if self.loss_function == "XiaoiceSing2":
+            log_f0 = pitch[:, : pitch_lengths.max()]
+            vuv = log_f0 != 0
```
Comment (ftshijt) [inline espnet2/svs/xiaoice/XiaoiceSing.py:489]: For log f0, why you are checking the 0 value? Isn't should be to check whether it is negative or positive?

**Review by ftshijt — COMMENTED** [review]
LGTM! please check the vuv part to make sure it is correct

File: espnet2/svs/xiaoice/XiaoiceSing.py (hunk 2 above)
Comment (A-Quarter-Mile) [reply espnet2/svs/xiaoice/XiaoiceSing.py:490]: Fixed it by setting negative log_f0 to zero.

**Pull request merged**